docs/ZendeskMultiObjects-batchsource.md

```diff
@@ -56,19 +57,25 @@ the specified date. The date must be provided in the date format:
 Specifying this along with `Start Date` allows reading data modified within a specific time window. 
 If no value is provided, no upper bound is applied.
 
-**Satisfaction Ratings Score:** Filter Satisfaction Ratings object to include only records which have Zendesk score
-equal to the specified score.
+**Satisfaction Ratings Score:** Filter Satisfaction Ratings object to include only records which have Zendesk score equal to the specified score. 
+Only Applicable for Satisfaction Ratings object. 
+The satisfaction ratings score cannot be a numerical range. 
```

Comment:
This is impossible to do in the UI. I don't think we should have this in the doc. There's actually no way to do this.

Reply:
removed